test/clj/game/cards/assets_test.clj

```diff
@@ -3447,40 +3432,40 @@
         (run-empty-server state :archives)
         (take-credits state :runner)
         (play-from-hand state :corp "Neural EMP")
-        ; TODO: Implement interrupts to make these prompts work correctly
-        ; (is (not-empty (:prompt (get-corp))) "Prana prompt for Corp")
-        ; (is (empty? (:prompt (get-runner))) "No Caldera prompt for Runner")
-        )))
-  (testing "Runner cards don't trigger Prana"
-    (do-game
-      (new-game {:corp {:hand ["Prāna Condenser" "Shock!"]}
-                 :runner {:deck [(qty "Sure Gamble" 5)]
-                          :hand ["Zer0" "Sure Gamble"]}})
-      (play-from-hand state :corp "Prāna Condenser" "New remote")
-      (let [pc (get-content state :remote1 0)]
-        (rez state :corp pc)
-        (take-credits state :corp)
-        (play-from-hand state :runner "Zer0")
-        (card-ability state :runner (get-hardware state 0) 0)
-        (is (empty? (:prompt (get-corp))) "Prana condenser doesn't proc from Zer0"))))
-  (testing "PAD Tap gains credits from Prana trigger. Issue #5250"
-    (do-game
-      (new-game {:corp {:hand ["Prāna Condenser" "Bio-Ethics Association"]}
-                 :runner {:deck [(qty "Sure Gamble" 5)]
-                          :hand [(qty "PAD Tap" 3)]}})
-      (play-from-hand state :corp "Prāna Condenser" "New remote")
-      (play-from-hand state :corp "Bio-Ethics Association" "New remote")
-      (let [pc (get-content state :remote1 0)
-            bio (get-content state :remote2 0)]
-        (rez state :corp pc)
-        (rez state :corp bio)
-        (take-credits state :corp)
-        (play-from-hand state :runner "PAD Tap")
-        (play-from-hand state :runner "PAD Tap")
-        (play-from-hand state :runner "PAD Tap")
-        (take-credits state :runner)
-        (click-prompt state :corp "Yes")
-        (is (= 9 (:credit (get-runner))) "Runner gained 3 credits from Prana")))))
+        (is (not-empty (:prompt (get-corp))) "Prana prompt for Corp")
+        (is (= :waiting (prompt-type :runner))))))
+    (testing "Runner cards and costs don't trigger Prana"
+      (do-game
+        (new-game {:corp {:hand ["Prāna Condenser" "Shock!"]}
+                   :runner {:deck [(qty "Sure Gamble" 5)]
+                            :hand ["Zer0" "Sure Gamble" "Sure Gamble"]}})
+        (play-from-hand state :corp "Prāna Condenser" "New remote")
+        (let [pc (get-content state :remote1 0)]
+          (rez state :corp pc)
+          (take-credits state :corp)
+          (play-from-hand state :runner "Zer0")
+          (card-ability state :runner (get-hardware state 0) 0)
+          (is (empty? (:prompt (get-corp))) "Prana condenser doesn't proc on 'unpreventable' net damage")
+          ;; (damage state :runner :net 1)
```

Comment:
Comments?

Reply:
attempting to deal net damage as the runner shouldn't prompt prana, but that isn't supported yet.